### ISSUE TITLE
Rst preview does not work with pasted code

### ISSUE THREAD

**Issue by thatrandomperson5** [retext-project/retext#589]
When I paste my code the preview won't load.

code:
```rst
#####
Configure Commanger: configure all the way!
#####

*****
Configuration tutorial
*****

*****
Types:
*****

Toggle
======

Long Toggle
^^^^^^

Char
======

Long
=====

Sticky
=====

Range
=====

Sticky Range
^^^^^

Long Range
^^^^^

Hard
=====
```

**Comment by paixi** [comment]
I also get an error with MD previews in files containing code blocks using ReText 8.0.0 (PyMarkups 3.1.3) in Python 3.10.7. It worked fine before in Python 3.9.2.
```
      File "/home/paixi/.local/lib/python3.10/site-packages/ReText/mdx_posmap.py", line 114, in run
        last_child = SubElement(parent, 'p')
    TypeError: SubElement() argument 1 must be xml.etree.ElementTree.Element, not Element
```
The problem can be fixed by changing [mdx_posmap.py:112](https://github.com/retext-project/retext/blob/fa18e915478014a2a66d70944d4e6ce417ce57ad/ReText/mdx_posmap.py#L112) to:
`last_child = xml.etree.ElementTree.SubElement(parent, 'p')`
And [line 19](https://github.com/retext-project/retext/blob/fa18e915478014a2a66d70944d4e6ce417ce57ad/ReText/mdx_posmap.py#L19) to:
`import xml`
Haven't had any issues after that.

**Comment by mitya57** [comment]
@paixi The issue that you were seeing is probably the same as one discussed in Python-Markdown/markdown#950.
It happens when defusedxml module is present, and fixed in tiran/defusedxml#60.

However, the original reporter’s issue is not necessarily the same one, however as I got no reply to it, I am closing this bug.
